src/main/resources/templates/fragments/header.html

```diff
@@ -18,27 +18,26 @@
         <link href="http://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700" rel="stylesheet" type="text/css"/>
         <link rel="shortcut icon" href="images/ncl_favicon1.png" th:href="@{~/images/ncl_favicon1.png}"/>
 
-        <!--[if lt IE 9]>
-        <script type="text/javascript" src="static/js/html5shiv.js" th:src="@{~/js/html5shiv.js}"></script>
-        <script type="text/javascript" src="static/js/respond.min.js" th:src="@{~/js/respond.min.js}"></script>
-        <![endif]-->
+        <!--&lt;!&ndash;[if lt IE 9]>-->
+        <!--<script type="text/javascript" src="static/js/html5shiv.js" th:src="@{~/js/html5shiv.js}"></script>-->
+        <!--<script type="text/javascript" src="static/js/respond.min.js" th:src="@{~/js/respond.min.js}"></script>-->
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)